queue/util/createConsumer.js

```diff
@@ -47,6 +47,8 @@ module.exports = async function (options) {
     } catch (err) {
       logger.error('Error handling message:', err)
 
+      // TODO: This needs more thought. If a message is continuously failing,
+      // we shouldn't keep requeuing
```

Comment:
I agree. I just saw this code as I was moving along and thought about this. That's why Ieft the comment.